io.github.webcamoid.Webcamoid.yml

```diff
@@ -0,0 +1,26 @@
+app-id: io.github.webcamoid.Webcamoid
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: webcamoid
+rename-icon: webcamoid
+rename-appdata-file: io.github.webcamoid.metainfo.xml
```

Comment:
you could name the file properly in the repository, and you'd not need to do that.